docs/quickstarts/aws.md

```diff
@@ -5,174 +5,226 @@ weight: 10
 Add a private key corresponding to one of the public keys specified in `ssh_pubkeys` to your `ssh-agent`:
 
 ```bash
 ssh-add ~/.ssh/id_rsa
 ssh-add -L
 ```
 
-Run the following command to create the cluster:
+Deploy the cluster:
 
 ```console
-lokoctl cluster apply
+lokoctl cluster apply -v
+```
+
+The deployment process typically takes about 15 minutes. Upon successful completion, an output
+similar to the following is shown:
+
 ```
-Once the command finishes, your Lokomotive cluster details are stored in the path you've specified
-under `asset_dir`.
+Your configurations are stored in ./assets
+
+Now checking health and readiness of the cluster nodes ...
 
+Node                             Ready    Reason          Message                            
+                                                                                             
+lokomotive-demo-controller-0       True     KubeletReady    kubelet is posting ready status    
+lokomotive-demo-pool-1-worker-0    True     KubeletReady    kubelet is posting ready status    
+lokomotive-demo-pool-1-worker-1    True     KubeletReady    kubelet is posting ready status    
+
+Success - cluster is healthy and nodes are ready!
+```
 ## Verification
 
-A successful installation results in the output:
+Use the generated `kubeconfig` file to access the cluster:
 
 ```console
-module.aws-myawscluster.null_resource.bootkube-start: Still creating... [1m50s elapsed]
-module.aws-myawscluster.null_resource.bootkube-start: Still creating... [2m0s elapsed]
-module.aws-myawscluster.null_resource.bootkube-start: Creation complete after 2m6s [id=5156996152315868880]
+export KUBECONFIG=$(pwd)/assets/cluster-assets/auth/kubeconfig
+kubectl get nodes
+```
+
+Sample output:
 
-Apply complete! Resources: 118 added, 0 changed, 0 destroyed.
+```
+NAME                            STATUS   ROLES    AGE   VERSION
+lokomotive-demo-controller-0      Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-0   Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-1   Ready    <none>   33m   v1.17.4
+```
 
-Your configurations are stored in /home/imran/lokoctl-assets/myawscluster
+Verify all pods are ready:
 
-Now checking health and readiness of the cluster nodes ...
+```console
+kubectl get pods -A
+```
 
-Node              Ready    Reason          Message
+Verify you can access httpbin:
 
-ip-10-0-39-75     True     KubeletReady    kubelet is posting ready status
-ip-10-0-39-78     True     KubeletReady    kubelet is posting ready status
-ip-10-0-39-29     True     KubeletReady    kubelet is posting ready status
-ip-10-0-12-241    True     KubeletReady    kubelet is posting ready status
-ip-10-0-12-244    True     KubeletReady    kubelet is posting ready status
-ip-10-0-12-249    True     KubeletReady    kubelet is posting ready status
+```console
+kubectl -n httpbin port-forward svc/httpbin 8080
 
-Success - cluster is healthy and nodes are ready!
+# In a new terminal.
+curl http://localhost:8080/get
```

Comment:
A public IP can be used: `httpbin.<cluster name>.<DNS zone>`.

Reply:
This is not needed as we don't have contour setup. We are trying to go for as minimal setup as possible

Reply:
I think we should have contour, it is just one line.